content/news/playtest-20151031.md

```diff
@@ -0,0 +1,173 @@
+<p markdown="1">
+Halloween is upon us, and the latest OpenRA playtest is full of new tricks and other treats!
+This build includes all of the major new features and balance changes that we have planned for our Christmas release, and it is going to be one of our biggest releases yet!  Check out the
+[full changelog](http://changelog.openra.net) for more details about the balance changes and new
+features.
+</p>
+<p markdown="1">
+We rely on your testing and feedback to make sure that our release builds are as balanced and bug-free as possible, so please let us know if you find any issues or just generally don't like something by [filing a bug](http://bugs.openra.net) or [joining our IRC channel](/community/)!  We will try our best to fix any
+issues before the final release.
+</p>
+<p markdown="1">
+Grab the installer now for your operating system from our [download page](/download/)!
+</p>
+
+
+<hr />
+## Ten reasons to try the new OpenRA Playtest
+<hr />
+### 1. New chat lobby
+
+<div style="text-align:center" markdown="1">
+![Global chat lobby](/images/news/20151031-multiplayer.png)
+</div>
+
+<p>
+The multiplayer server browser and game lobby have been redesigned around a new global
+chat channel.<br />This will make it easier to find and talk with other players, and to organize matches without server-hopping.
+</p>
+<p>
+The channel is available from outside the game as well, by connecting to
+irc.openra.net:6667 channel #lobby using your favourite IRC client.
```

Comment:
Is it worth turning this into a URL? Not sure how widespread support for irc:// is.